Add Caution base on Lighthouse changes (#16043)
mdn/content: files/en-us/glossary/first_meaningful_paint/index.md

  - Reference
  - Web Performance
---

> > **Caution:** First Meaningful Paint (FMP) is deprecated in Lighthouse 6.0. In practice FMP has been overly sensitive to small differences in the page load, leading to inconsistent (bimodal) results. Additionally, the metric's definition relies on browser-specific implementation details, which means it cannot be standardized nor implemented in all web browsers. Moving forward, consider using Largest Contentful Paint instead.

Reviewer: ```suggestion
> **Warning:** First Meaningful Paint (FMP) is deprecated in Lighthouse 6.0.  Moving forward, consider using [Largest Contentful Paint](https://wicg.github.io/largest-contentful-paint/) instead.
```

**First Meaningful Paint** (FMP) is the paint after which the biggest above-the-fold layout change has happened and web fonts have loaded.  It is when the answer to "Is it useful?" becomes "yes", upon first meaningful paint completion.

> 

Reviewer: FMP is very sensitive to small differences in the page load. This can lead to inconsistent (bimodal) results. The metric's definition relies on browser-specific implementation details, which means it can't be standardized and hand not been implemented in all web browsers.

Author: Thank you @estelle

  - Reference
  - Web Performance
---

> **Warning:** First Meaningful Paint (FMP) is deprecated in Lighthouse 6.0.  Moving forward, consider using [Largest Contentful Paint](https://wicg.github.io/largest-contentful-paint/) instead.

**First Meaningful Paint** (FMP) is the paint after which the biggest above-the-fold layout change has happened and web fonts have loaded.  It is when the answer to "Is it useful?" becomes "yes", upon first meaningful paint completion.

> FMP is very sensitive to small differences in the page load. This can lead to inconsistent (bimodal) results. The metric's definition relies on browser-specific implementation details, which means it can't be standardized and hand not been implemented in all web browsers.

Reviewer 2: ```suggestion
FMP is very sensitive to small differences in the page load. This can lead to inconsistent (bimodal) results. The metric's definition relies on browser-specific implementation details, which means it can't be standardized and has not been implemented in all web browsers.
```
